src/Toml/Bi/Combinators.hs

```diff
@@ -293,6 +305,58 @@ arrayNonEmptyOf :: TomlBiMap a AnyValue -> Key -> TomlCodec (NonEmpty a)
 arrayNonEmptyOf = match . _NonEmpty
 {-# INLINE arrayNonEmptyOf #-}
 
+----------------------------------------------------------------------------
+-- Monoid codecs
+----------------------------------------------------------------------------
+
+{- | Codec for 'All' wrapper for boolean values.
+
+@since 1.2.1.0
+-}
+all :: Key -> TomlCodec All
+all = dimap (Just . getAll) (All . fromMaybe False) . dioptional . bool
+{-# INLINE all #-}
+
+{- | Codec for 'Any' wrapper for boolean values.
+
+@since 1.2.1.0
+-}
+any :: Key -> TomlCodec Any
+any = dimap (Just . getAny) (Any . fromMaybe False) . dioptional . bool
```

Comment:
That's an interesting idea! My initial thought was that this codec should always require a boolean field. But actually quite often I want to have missing fields that defaults to `False` (especially in `summoner`). And since `any` is a `Monoid`, it's actually quite fine to return `False` on missing fields, great idea :clap: 

Should we add to documentation that returns `False` on missing fields?